shared/js/background/storage/https.es6.js

```diff
@@ -22,6 +25,15 @@ class HTTPSStorage {
             const etag = settings.getSetting(`${listCopy.name}-etag`) || ''
 
             return this.getDataXHR(listCopy.url, etag).then(response => {
+                // Set the lastUpdate time.
+                // Note: Take the earliest time between server and local, that way
+                //       if the local time is set far in the future updates will
+                //       still happen.
+                const localTime = Date.now()
+                const serverTime = Date.parse(response.date)
+                const updateTime = Math.min(localTime, serverTime || localTime)
```

Comment:
Can you mention here that the `|| localTime` is to remove `NaN` should the date now parse (as otherwise `Math.min` will pick `NaN`). This is a gotcha which I think should be pointed out to prevent a regression if future maintainers think they can refactor this.

Reply:
Good idea, Done.